src/frameworks/opengles/gles_guest.rs

```diff
@@ -678,6 +732,20 @@ fn glDeleteRenderbuffersOES(env: &mut Environment, n: GLsizei, renderbuffers: Mu
     })
 }
 
+fn glBufferData(env: &mut Environment, target: GLenum, n: GLsizei, data: ConstVoidPtr, usage: GLenum) {
+    with_ctx_and_mem(env, |gles, mem| {
+        let data_ptr = mem.ptr_at(data.cast::<u8>(), 0).cast::<GLvoid>();
+        let n_usize: GLsizeiptr = n.try_into().unwrap();
+        unsafe { gles.BufferData(target, n_usize, data_ptr, usage) }
+    })
+}
```

Comment:
These other functions look fine, but:

* Could you put them in separate commits from the texturing-related changes, since they seem unrelated?
* Could you try to put them near related functions in the file, and try to do this in all three GLES files? I notice that their position in the file doesn't match their position in the list of exported functions. Btw I think it'd make sense if `glBufferData` goes with the other functions related to buffers.